tests/page-objects/default/enketo/undo-death-report.page.js

```diff
@@ -0,0 +1,14 @@
+const FORM ='form[data-form-id="undo_death_report"]';
+
+const submitReporConfirmation = (value) => $(`${FORM} ` +
```

Comment:
Typo `Repor`
The "submit" word got me confused when reading the code below, we're not submitting the report yet, right? 
What about naming it `confirmUndoDeath` or `confirmUndoDeathOption` since it retrieves one radio element?

If it doesn't fit in the line, there are these options:

OP 1: selector in one line - allows you to colapse the block in the IDE, if you need. 
```
const confirmUndoDeathOption = (value) => {
  return $(`${FORM} input[name="/undo_death_report/undo/undo_information"][value="${value}"]`);
};
```

OP 2: unwrap form constant
```
const confirmUndoDeathOption = (value) => $(FORM +
  ` input[name="/undo_death_report/undo/undo_information"][value="${value}"]`);
```

OP 3: block style
```
const confirmUndoDeathOption = (value) => $(
  FORM +
  ` input[name="/undo_death_report/undo/undo_information"][value="${value}"]`
);
```

Reply:
you can apply the one you chose in `death-report.page.js` selectors too :)

Reply:
Thank you for these suggestions Jenny!